packages/testing/scripts/download-test-server.mjs

```diff
@@ -24,6 +24,20 @@ const defaultHeaders = {
   'User-Agent': '@temporalio/testing installer',
 };
 
+const { GITHUB_TOKEN } = process.env;
+if (GITHUB_TOKEN) {
+  console.log(`Using GITHUB_TOKEN`);
+  defaultHeaders['Authorization'] = `Bearer ${GITHUB_TOKEN}`;
+}
+
+const rateLimitResult = await got('https://api.github.com/rate_limit', {
```

Comment:
Let's not put this here, it's nice for CI but we don't want this for every user installing the package